pkg/virt-handler/virt-chroot/virt-chroot.go

```diff
@@ -32,16 +34,28 @@ const (
 	mountNamespace = "/proc/1/ns/mnt"
 )
 
+// GetHostNamespacePath - return the parent process
```

Comment:
Please update the comment here to match the code:
```suggestion
// GetHostNamespacePath - return the path of the parent process mount namespace
```

Reply:
got it. pushed in the fix.